### ISSUE TITLE
`agda-bisect` does not check commandline for test case

### ISSUE THREAD

**Issue by andreasabel** [agda/agda#6447]
`agda-bisect` currently does not alert you if you forget to supply a test to run bisect against.  I.e., it starts running even if none of the following are given:
```
  ARGUMENTS...             The arguments supplied to Agda
  --script PROGRAM         Do not invoke Agda directly, run PROGRAM instead
  PROGRAM ARGUMENTS...     Extra arguments for the --script program
```
This may cost the user many minutes for nothing...

**Comment by nad** [comment]
It is perhaps not inconceivable that someone might want to test the command `agda` without any arguments.

**Comment by andreasabel** [comment]
> someone might want to test the command `agda` without any arguments.

I'd say this feature isn't worth preserving, or one could add some flag that would allow that.  But I'd rather get a meteorite on my head than needing this feature (probability wise).

**Comment by nad** [comment]
One example: We realise that at some point `agda` without arguments started segfaulting on some platform that we don't test regularly.

I'm guessing that you proposed this change because you used the tool incorrectly. However, there are many ways to do this, and I don't think I've encountered this one before. Instead of the proposed change I recommend you to use `--dry-run` to ensure that the `--good` and `--bad` cases behave as intended before you start a possibly expensive bisection. One could change `agda-bisect` so that this is taken care of automatically, but this might involve building Agda twice, and often one already has builds of Agda corresponding to the `--good` and `--bad` commits.

**Comment by andreasabel** [comment]
Yes, I think it makes sense that the dry-run is a manual process.  I usually to a `--dry-run-with-commit` which might take around 15min (if many dependencies need to be installed).  It is good to not do this "emptily".


> One example: We realise that at some point `agda` without arguments started segfaulting on some platform that we don't test regularly.

```console
$ touch Foo.agda
$ agda Foo.agda
(succeeds)
```
A `touch` isn't a lot of effort, I say.  Covers this exotic "meteorite-fall-on-my-head" case. ;-)
